packages/linea-state-verifier/contracts/EVMFetcher.sol

```diff
@@ -230,6 +230,10 @@ library EVMFetcher {
      * @dev Initiates the fetch request.
      *      Calling this function terminates execution; clients that implement CCIP-Read will make a callback to
      *      `callback` with the results of the operation.
+     * @dev This implementation makes use of callbackData to pass the acceptedL2BlockRangeLength needed to check
+     *      the block used when checking the proof is not too old, this is a difference from ENS initial implementation
```

Comment:
`that the block used`

Maybe note what the ENS implementation does differently